.storybook/main.ts

```diff
@@ -3,18 +3,20 @@ import type {StorybookConfig} from '@storybook/react-webpack5';
 
 import pkg from '../package.json';
 
-const config:StorybookConfig = {
+const config: StorybookConfig = {
     framework: {
         name: '@storybook/react-webpack5',
-        options: {fastRefresh: true},
```

Comment:
Why was removed `fastRefresh`?

Reply:
Because they don't have that option right now.
As I understand it, this is now the default